### PR TITLE
didu: init at 2.5.1

### PR DESCRIPTION
**Pull request by auroraanna**
###### Description of changes

Add the didu package

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by papojari** [comment]
Wheew… sorry for updating all the time. I'm the developer of didu and have been finding ways to make the program better. After one version I always thought that it would be the last one for now. I think this was the last update though.

**Comment by papojari** [comment]
That aged like milk.

**Comment by bobby285271** [comment]
@ofborg eval

**Comment by bobby285271** [comment]
Please fix the hash mismatch, I think you forget to update that on your [last force-push](https://github.com/NixOS/nixpkgs/compare/0435d0eaa413e91a51e995a869f89e2ee311a2aa..6e9a1de624bd86a7608648981ec7f3efc54574eb)

File: pkgs/tools/misc/didu/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  cargoSha256 = "sha256-eXke4G7fIRfKuyGXZ7izOtN1xDFP5eqmTq1+5os0pTI=";
+
+  meta = with lib; {
+    description = "Duration conversion between units";
+    homepage = "https://codeberg.org/papojari/didu";
+    license = licenses.lgpl3Only;
+    maintainers = with maintainers; [ papojari ];
+  };
+}
+
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/didu/default.nix:24]: one final new line to much

**Pull request merged**